private-templates-service/client/src/components/Designer/SaveModal/SaveModal.tsx

```diff
@@ -131,7 +131,7 @@ class SaveModal extends React.Component<Props, State> {
               </Container>
               <InfoWrapper>
                 <StyledH3>{CARDNAME}</StyledH3>
-                <StyledTextField onChange={this.onChange} />
+                <StyledTextField onChange={this.onChange} placeholder = {UNTITLEDCARD} />
```

Comment:
`placeholder` sets the greyed out unselectable text inside the textfield, `defaultValue` sets the initial value of the textfield.

IMO, `defaultValue={UNTITLEDCARD}` and `placeholder={"My Card"}`, something like that.